plugins/dynamix/scripts/tty_font

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+if [[ -n $1 ]]; then
+  pkill ttyd
+  sed -ri "s/fontSize=[0-9]+/fontSize=$1/" /etc/default/ttyd
```

Comment:
Is `#arg[1]` escaped? If not when it's passed down to `$1` code injection is possible.

Reply:
arguments are automatically escaped in update.php which prepares the request